url-shortner-view/src/View/View.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
-import classes from './View.module.css';
 import FirstView from '../FirstView/FirstView';
 import Features from '../Features/Features';
-import Header from '../Header/Header';
-const View = () => (
-	<main>
-		<FirstView />
-		<Features />
-	</main>
+
+const View = (props) => (
+  <main>
+    <FirstView
+      url={props.url}
+      setUrl={props.setUrl}
+      shortUrl={props.shortUrl}
+      setShortUrl={props.setShortUrl}
+    />
+    <Features />
+  </main>
```

Comment:
since view is only adding main tag and displaying FirstView, You can discard View component and use FirstView directly in App.js. Then you can add main tag INSIDE first view

Reply:
View is displaying FirstView and Features not only FirstView